docs/auth.md

```diff
@@ -103,3 +113,19 @@ Like the other OAuth2 providers, access to Chronograf via Heroku can be restrict
 ```sh
 export HEROKU_ORGS=hill-valley-preservation-sociey,the-pinheads
 ```
+
+### Optional: Configuring Authentication Duration
+
+By default, auth will remain valid for 30 days via a cookie stored in the browser. This duration can be changed with the environmental variable `AUTH_DURATION`. For example, to change it to 1 hour, use:
```

Comment:
This is an interesting point -- good catch. Per discussion with @goller, we feel like someone who's reading auth docs _will_ care about how security is implemented, for a multitude of reasons. This helps them make decisions about the authentication duration for their particular setup. So we feel like it's important to leave this be for now.